packages/flutter_chat/lib/src/flutter_chat_navigator_userstory.dart

```diff
@@ -178,11 +200,25 @@ Widget _newChatScreenRoute(
       options: configuration.chatOptionsBuilder(context),
       translations: configuration.translations,
       service: configuration.chatService,
+      onPressCreateGroupChat: () async {
+        if (configuration.onPressCreateGroupChat != null) return;
```

Comment:
That function call has to be there right? I just didn't understand what you did here